project/ConsoleTracker/ConsoleTrackerApp/tasks.py

```diff
@@ -27,6 +31,50 @@ def switch_on(ip):
 	t = Thread(target=turn_on_switch_help, args=(ip,))
 	t.start()
 
+# runs the update time function 
+def update_time(socket, username, time):
+    return socket.update_time(username, time)
+
+#thread to update the time balance of the user
+def update_time_thread():
+    #represents the seconds on the clock we will update time
+    check_seconds = [00, 15, 30, 45]
+    interval = 15
+    #create the socket that connects to ExternalSocket
+    socket = InternalSocket()
+
+    while True:
+        #The current system time and the seconds of the current clock
+        curr_time = datetime.datetime.now().replace(tzinfo=pytz.UTC)
```

Comment:
```suggestion
        curr_time = timezone.now()
```
getting weird numbers with the original
![image](https://user-images.githubusercontent.com/59661596/151855149-d5cd60fe-c9fb-4130-8b03-9d4ded5a83a6.png)
using the same django's timezone class used for machine.start_time gives normal looking numbers
![image](https://user-images.githubusercontent.com/59661596/151855233-0ea623e7-fe2a-4546-b2ec-072ff0361a4a.png)